gulpfile.js

```diff
@@ -44,9 +44,14 @@ gulp.task('watch-css', () => {
 const fontFiles = 'h/static/styles/vendor/fonts/h.woff';
 
 gulp.task('build-fonts', () => {
+  const cwd = 'h/';
   const fontsDir = 'build/fonts';
   return gulp
-    .src(fontFiles, { encoding: false })
+    .src(fontFiles.slice(cwd.length), {
+      encoding: false,
+      // Prevent `gulp.src` from attempting to stat files outside of `cwd`
+      cwd,
+    })
```

Comment:
Since the same logic is defined twice, perhaps we could create a local helper that defines the common parts:

```js
const cwd = 'h/';
const gulpSRC = globs => gulp.src(globs.slice(cwd.length, {
  // Treat all files as binary. Some of the images are SVGs which are text,
  // but `svgmin` is still able to process the files if passed as binary.
  encoding: false,
  // Prevent `gulp.src` from attempting to stat files outside of `cwd`
  cwd,
});

// ...

return gulpSRC(fontFiles)
  .pipe(changed(fontsDir))
  .pipe(gulp.dest(fontsDir));

// ...

return gulpSRC(imageFiles)
  .pipe(changed(imagesDir))
  .pipe(gulpIf(shouldMinifySVG, svgmin(svgminConfig)))
  .pipe(gulp.dest(imagesDir));
```

Reply:
I'm going to merge this as-is to unblock CI, but I'd like to find a way to remove the footguns that the `encoding` and `cwd` options work around.